clients/spark/core/src/main/scala/io/treeverse/clients/ExportStatus.scala

```diff
@@ -0,0 +1,4 @@
+package io.treeverse.clients
+
+// ExportStatus is the export status of a single file
+case class ExportStatus(path: String, success: Boolean, msg: String) extends Serializable {}
```

Comment:
Ewww... That's a Go comment... 
Consider /** ... */. And it needs the imperative mode, without repeating the function name. 
🤷